teacher.rb

```diff
@@ -1,14 +1,15 @@
+# Teacher Class
 require_relative 'person'
 
 class Teacher < Person
   attr_accessor :specialization
 
-  def initialize(id, specialization, name = "Unknown", age = 0)
+  def initialize(id, specialization, name = 'Unknown', age = 0)
```

Comment:
- [ ] Kindly add the same update to the parameters as done in the student and person class 👍